Dockerfile

```diff
@@ -13,8 +13,10 @@ ARG PYTORCH_IMAGE=nvcr.io/nvidia/pytorch:20.10-py3
 
 FROM ${PYTORCH_IMAGE} as base
 
+MAINTAINER Wenqi Li <wenqil@nvidia.com>, Nic Ma <nma@nvidia.com>, Daniel Schulz <danielschulz2005@hotmail.com>
```

Comment:
this could be consistent with https://github.com/Project-MONAI/MONAI/blob/115f146e4e37e23c7241ac5decfec42bb7aa6c84/setup.cfg#L2-L10